src/Consolonia.NUnit/UnitTestConsole.cs

```diff
@@ -138,7 +137,7 @@ public async Task WaitRendered()
         {
             await Dispatcher.UIThread.InvokeAsync(async () =>
             {
-                Window mainWindow = _lifetime.MainWindow!;
+                var mainWindow = _lifetime.TopLevel as Window;
                 mainWindow!.InvalidateVisual();
```

Comment:
From the error message will not be possible to understand if TopLevel is null or if it is not a Window.